src/main/java/gregtech/api/items/toolitem/IGTTool.java

```diff
@@ -144,9 +144,31 @@ default ItemStack get(Material material) {
         // Set Material
         toolTag.setString(MATERIAL_KEY, material.toString());
 
+        // Grab the definition here because we cannot use getMaxAoEDefinition as it is not initialized yet
+        AoESymmetrical aoeDefinition = getToolStats().getAoEDefinition(stack);
+
         // Set other tool stats (durability)
         ToolProperty toolProperty = material.getProperty(PropertyKey.TOOL);
-        toolTag.setInteger(MAX_DURABILITY_KEY, toolProperty.getToolDurability());
+
+        // Multiply the durability based on the maximum AOE definition of the tool
+        int maxAOEArea = 1;
+        if (aoeDefinition != AoESymmetrical.none()) {
+            maxAOEArea = Math.max(aoeDefinition.column, Math.max(aoeDefinition.row, aoeDefinition.layer));
+        }
+
+        int durability = toolProperty.getToolDurability() * toolProperty.getDurabilityMultiplier();
+
+        // Most Tool Definitions do not set a base durability, which will lead to ignoring the multiplier if present. So apply the multiplier to the material durability if that would happen
+        if (toolStats.getBaseDurability(stack) * toolStats.getDurabilityMultiplier(stack) == 0) {
```

Comment:
You are going to use `toolStats.getDurabilityMultiplier` anyway so there is no need to have it in `if`